WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -164,26 +162,6 @@ private extension SettingsViewController {
         }
     }
 
-    func rowsForImproveTheAppSection(onCompletion: @escaping (_ rows: [Row]) -> Void) {
-        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productList) {
-            onCompletion([.privacy, .betaFeatures, .featureRequest])
-            return
-        }
-
-        guard let siteID = ServiceLocator.stores.sessionManager.defaultStoreID else {
-            assertionFailure("Cannot find store ID")
-            return
-        }
-        let action = AppSettingsAction.loadStatsVersionEligible(siteID: siteID) { eligibleStatsVersion in
```

Comment:
maybe we can still keep the check for stats v4 eligibility before we drop stats v3 (https://github.com/woocommerce/woocommerce-ios/issues/1946 https://github.com/woocommerce/woocommerce-ios/issues/1868)?

Reply:
I'd say yes, but in this specific case, with the `productList` enabled the stats v4 eligibility was never called. Do you think it would be worth removing just the check for the feature flag?

Reply:
after thinking about the settings again, removing this check is good since we're adding a Products feature switch for this release so the `betaFeatures` row is always shown 👍